### PR TITLE
Do ABSEIL + Protobuf Migration

### PR DESCRIPTION
**Pull request by hmaarrfk**
At the request of @BastianZim a little more time is given.

cc: @h-vetinari 

xref: https://github.com/conda-forge/or-tools-feedstock/pull/24#issuecomment-1242766384

Closes: https://github.com/conda-forge/or-tools-feedstock/pull/26

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml
Comment (BastianZim) [inline recipe/meta.yaml:4]: I would like to keep this in case we ever need that name.

File: recipe/meta.yaml
Comment (BastianZim) [inline recipe/meta.yaml:38]: This was added because it is the only version that was supported as newer versions had a bug, if I remember correctly. As pointed out, they don't get propagated to run though, so this might need to get fixed.

@tkralphs Sorry to be pinging you of the blue but do you know if everything is resolved with or-tools now and we can remove this?

**Review by BastianZim — CHANGES_REQUESTED** [review]
One small thing and one question for a dependency. Otherwise, I'm happy to merge this. Thanks, everyone. :)

File: recipe/meta.yaml
Comment (h-vetinari) [reply recipe/meta.yaml:4]: In an multi-output recipe, the name of the "package" is irrelevant and doesn't show up anywhere. But fine by me to re-add.

Comment (h-vetinari) [inline recipe/meta.yaml:4]:
```suggestion
  name: or-tools-packages
```

Comment (hmaarrfk) [inline recipe/meta.yaml:4]:
```suggestion
  name: or-tools-package
```
No it is useful to have this be a different names

Comment (hmaarrfk) [reply recipe/meta.yaml]: sorry, i did the same at the same time.

File: recipe/meta.yaml
Comment (hmaarrfk) [reply recipe/meta.yaml:38]: > As pointed out, they don't get propagated to run though, so this might need to get fixed.

What is getting propagated is "1.17" not "1.17.4"
https://github.com/conda-forge/coin-or-clp-feedstock/pull/5/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR18

File: recipe/meta.yaml
Comment (hmaarrfk) [inline recipe/meta.yaml:83]:
```suggestion
        - coin-or-clp =1.17.4
```
Lets try

File: recipe/meta.yaml
Comment (BastianZim) [reply recipe/meta.yaml:38]: Yes which is probably a mistake but I cannot remember fully what the situation was. I would prefer to wait for Ted and then merge but I can also ask or-tools otherwise.

File: recipe/meta.yaml
Comment (BastianZim) [reply recipe/meta.yaml]: If I remember correctly versions 1.17.5 and 1.17.6 were broken with or-tools which is why this is introduced. And you're right, it should've been added to run as well.

File: recipe/meta.yaml
Comment (BastianZim) [reply recipe/meta.yaml]: Yes, more about keeping the recipe clean should the name ever be used.

File: recipe/meta.yaml
Comment (hmaarrfk) [reply recipe/meta.yaml:38]: Understood

File: recipe/meta.yaml
Comment (h-vetinari) [reply recipe/meta.yaml]: > If I remember correctly versions 1.17.5 and 1.17.6 were broken with or-tools which is why this is introduced.

What about the current 1.17.7?

Comment (BastianZim) [reply recipe/meta.yaml]: > What about the current 1.17.7?

That wasn't available at the time, so not sure. That's why I would like to ask Ted first.

File: recipe/meta.yaml
Comment (h-vetinari) [reply recipe/meta.yaml]: > As pointed out, they don't get propagated to run though, so this might need to get fixed.

You can add `coin-or-clp !=1.17.5,!=1.17.6 to the run requirements (though builds against 1.17.7 would automatically get that lower bound from the run export)

Comment (BastianZim) [reply recipe/meta.yaml]: @tkralphs Gentle ping if you have a second.

Comment (BastianZim) [reply recipe/meta.yaml]: Merging this now since we didn't hear from upstream. I guess we can always patch the metadata if we get issues.

**Comment by BastianZim** [comment]
Thanks everyone!

File: recipe/meta.yaml
Comment (BastianZim) [reply recipe/meta.yaml]: Actually, since he's not available and you're also the maintainer of most these packages, should we remove it and see what happens? @h-vetinari

Comment (h-vetinari) [reply recipe/meta.yaml]: Yeah, feel free to remove it from my POV

**Pull request merged**
